usage/src/main/resources/org/killbill/billing/usage/ddl.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE rolled_up_usage (
     unit_type varchar(255),
     record_date date NOT NULL,
     amount bigint NOT NULL,
+    tracking_id varchar(128) NOT NULL,
```

Comment:
Could you also update the [wiki](https://github.com/killbill/killbill/wiki/Kill-Bill-Migration-0.16.x-to-0.18.x) with the `ALTER` statement for the migration?